content/building-blocks/store-and-document-your-data/store-data/environment-variables.md

```diff
@@ -32,7 +34,7 @@ Never upload a script that contains privacy-sensitive information to Github. Eve
 2. In the left sidebar click on "Advanced system settings".
 3. Click on "Environment Variables" in the bottom right.
 4. Create a new "User Variable" (top list) and fill out the "Variable name" and "Variable value".
-5. Double click "OK" twice.
+5. Double-click "OK" twice.
 
 
```

Comment:
Add a very brief "summary" section with key bullet points in the end maybe?

Reply:
Good point, I just added it at the end right before "see also" 🎯

Reply:
PD: I've seen that in other building blocks the final "Summary" section is below the "See also" one so I moved it to that position.